.github/workflows/aur-nym-vpn-x.yml

```diff
@@ -0,0 +1,56 @@
+# Publish vpn-x on AUR
+
+name: aur-nym-vpn-x
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: "Tag name of the release"
+        required: true
+        default: nym-vpn-x-v0.1.0
+      pkgver:
+        description: "PKGBUILD package version (should match the release version)"
+        required: true
+        default: 0.1.0
```

Comment:
The toolchain files?

Reply:
The cargo or typescript config files, from which you can fetch the version

Reply:
The motivation initially is to keep the possibility to set the arch package version to something different from the app version (even if we should aim to use the same version of the app).
Because Arch pkg version has some naming restriction, for example it can include hyphen `-` (we are using hyphen a lot in release versions in the repo)

ref https://wiki.archlinux.org/title/PKGBUILD#pkgver